darkmoon/app.py

```diff
@@ -17,7 +26,65 @@ def get_app(settings: Settings | None = None) -> FastAPI:
 
     app = FastAPI()
 
+    @app.exception_handler(MissingHashException)
+    async def missingHashExcHndlr(rqt: Request, exc: MissingHashException) -> jrp:
+        return jrp(
+            status_code=400,
+            content={"detail": "Enter Hash"},
+        )
+
+    @app.exception_handler(MissingHashTypeException)
+    async def msngHashTpExcHndlr(rqt: Request, exc: MissingHashTypeException) -> jrp:
+        return jrp(
+            status_code=400,
+            content={"detail": "Enter hash type"},
+        )
+
+    @app.exception_handler(InvalidIDException)
+    async def invalidIDExcHndlr(rqt: Request, exc: InvalidIDException) -> jrp:
+        return jrp(
+            status_code=400,
+            content={
+                "detail": (
+                    "This is not a valid ID",
+                    "It must be a 12-byte input or a 24-character hex string.",
+                ),
+            },
+        )
+
+    @app.exception_handler(ServerNotFoundException)
```

Comment:
This seems like an excessive solution for setting up your Exceptions. What I'm looking for is to utilize the HTTPException model and create Exceptions that can be grouped by similar types. This ticket is for code clarity, we don't need this.